crates/ra_hir_def/src/find_path.rs

```diff
@@ -185,69 +203,86 @@ fn select_best_path(old_path: ModPath, new_path: ModPath, prefer_no_std: bool) -
     }
 }
 
-fn find_importable_locations(
+/// Finds locations in `from.krate` from which `item` can be imported by `from`.
+fn find_local_import_locations(
```

Comment:
Long term, I imagine we might want some kind of index for this case as well.

Reply:
Yeah, sounds reasonable. Do you have a particular structure in mind? Which paths are applicable depends on the importing module here, so it's not as simple as `ImportMap`.

Reply:
I think something like `FxHashMap<ModuleDefId, Vec<(LocalModuleId, Visibility)>>` should work. Ie, we'd still have to compute the impourt ourselves, but we'd know the set of modules an item is visible at.

Reply:
To write down the second half of the original comment:

"... because we want to use this for fuzzy symobol index, and there we need to know private stuff"